dev/integration_tests/ios_host_app/FlutterUITests/FlutterUITests.m

```diff
@@ -126,6 +126,37 @@ - (void)testDualCold {
     XCTAssertTrue([app.navigationBars[@"Flutter iOS Demos Home"] waitForExistenceWithTimeout:kStandardTimeOut]);
 }
 
+- (void)testResize {
+  XCUIApplication *app = self.app;
+
+  [self waitForAndTapElement:app.buttons[@"Dynamic Content Resizing"]];
+
+  XCUIElementQuery *buttonQuery = [app.buttons matchingIdentifier:@"Add to list"];
+  [self expectationForPredicate:[NSPredicate predicateWithFormat:@"count = 1"] evaluatedWithObject:buttonQuery handler:nil];
+  [self waitForExpectationsWithTimeout:30.0 handler:nil];
+
+  XCUIElement *flutterView = app.otherElements[@"flutter_view"];
+
+  CGRect flutterViewFrame = flutterView.frame;
+  CGSize flutterViewSize = flutterViewFrame.size;
+
+  XCTAssertTrue(flutterViewSize.height == 142);
```

Comment:
Consider adding a comment as to how this size was determined. I imagine it's from manually examining the output of `dev/integration_tests/ios_host_app/flutterapp/lib/resize`? It might be worth adding link to this file in a the comment.